app/components/primer/alpha/x_banner.ts

```diff
@@ -1,8 +1,39 @@
 import {controller, target} from '@github/catalyst'
+// eslint-disable-next-line import/no-unresolved
+import '@primer/live-region-element/define'
```

Comment:
How to fix this? Also, I noticed we don't need to make this call in other custom elements since it's exported as part of the index. Wonder if we should do the same for `live-region`?